samples/simple-fsharp-console/FSharpConsole/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.Formatting;
+using Microsoft.DotNet.Interactive.FSharp;
+using System;
+using System.Linq;
+
+var kernel = new FSharpKernel()
+                    .UseDefaultFormatting()
+                    .UseNugetDirective()
+                    .UseKernelHelpers()
+                    .UseWho()
+                    .UseDotNetVariableSharing();
```

Comment:
Variable sharing won't provide any useful functionality without having more kernels present, so you can omit this.